gax/src/test/java/com/google/api/gax/rpc/RetryingTest.java

```diff
@@ -144,11 +139,7 @@ public void retryMaxAttemptsExceeded() {
         .thenReturn(RetryingTest.<Integer>immediateFailedFuture(throwable))
         .thenReturn(ApiFutures.<Integer>immediateFuture(2));
 
-    RetrySettings retrySettings = FAST_RETRY_SETTINGS.toBuilder().setMaxAttempts(2).build();
-    UnaryCallSettings<Integer, Integer> callSettings = createSettings(retrySettings);
-    UnaryCallable<Integer, Integer> callable =
-        FakeCallableFactory.createUnaryCallable(callInt, callSettings, clientContext);
-    callable.call(1);
+    assertRetrying(FAST_RETRY_SETTINGS.toBuilder().setMaxAttempts(2).build());
```

Comment:
The original test did not do assert (only did `callable.call(1)`). Was it a bug/missing assert in a test? If no, why expected result is changed here?

Reply:
I believe not having asserts must be intentional. As there was no need to assert the output we only needed to verifying if retrying happened or not.

> why expected result is changed here?

the expected result is common in 6 of existing test cases, So I extracted those lines. Please let me know if we should leave them as it is.